desktop-widgets/divelogexportdialog.ui

```diff
@@ -279,20 +279,26 @@
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="enabled">
-             <bool>false</bool>
+             <bool>true</bool>
             </property>
             <property name="title">
-             <string>CSV units</string>
+             <string>Options</string>
             </property>
             <widget class="QComboBox" name="CSVUnits_2">
+             <property name="enabled">
+              <bool>false</bool>
+             </property>
              <property name="geometry">
               <rect>
-               <x>30</x>
-               <y>30</y>
+               <x>10</x>
+               <y>20</y>
```

Comment:
above this line there are a few changes here which I'm not sure about... are these related to this PR?